backend/src/test/java/codezap/template/repository/SourceCodeRepositoryTest.java

```diff
@@ -0,0 +1,209 @@
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿과 소스코드 순서로 조회")
+    class FetchByTemplateAndOrdinal {
+        @Test
+        @DisplayName("성공: 템플릿과 소스코드 순서로 조회")
+        void fetchByTemplateAndOrdinalSuccess() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+            var template2 = templateRepository.save(new Template(member1, "Template 2", "Description 2", category1));
+            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", 1);
+            var sourceCode2 = new SourceCode(template1, "SourceCode 2", "Content 2", 2);
+            var sourceCode3 = new SourceCode(template2, "SourceCode 3", "Content 3", 1);
+            sut.save(sourceCode1);
+            sut.save(sourceCode2);
+            sut.save(sourceCode3);
+
+            var result = sut.fetchByTemplateAndOrdinal(template1, 1);
+
+            assertThat(result).isEqualTo(sourceCode1);
+        }
+
+        @Test
+        @DisplayName("실패: 잘못된 순서로 조회하는 경우 예외가 발생")
+        void fetchByTemplateAndOrdinalFaitWithWrongOrdinal() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", 1);
+            sut.save(sourceCode1);
+
+            var ordinal = 100;
+            assertThatThrownBy(() -> sut.fetchByTemplateAndOrdinal(template1, ordinal))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessageContaining("템플릿에 " + ordinal + "번째 소스 코드가 존재하지 않습니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿과 순서에 해당하는 전체 소스코드 조회")
+    class FindAllByTemplateAndOrdinal {
+        @Test
+        @DisplayName("성공: 템플릿과 순서에 해당하는 전체 소스코드 조회")
+        void findAllByTemplateAndOrdinalSuccess() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+            var template2 = templateRepository.save(new Template(member1, "Template 2", "Description 2", category1));
+            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", 1);
+            var sourceCode2 = new SourceCode(template1, "SourceCode 2", "Content 2", 1);
+            var sourceCode3 = new SourceCode(template1, "SourceCode 3", "Content 3", 2);
+            var sourceCode4 = new SourceCode(template2, "SourceCode 4", "Content 4", 1);
```

Comment:
ordinal인 1을 변수로 빼면 어떨까요? 더 가독성이 좋아질 것 같아요!
```suggestion
            int ordinal = 1;
            
            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", ordinal);
            var sourceCode2 = new SourceCode(template1, "SourceCode 2", "Content 2", ordinal);
            var sourceCode3 = new SourceCode(template1, "SourceCode 3", "Content 3", 2);
            var sourceCode4 = new SourceCode(template2, "SourceCode 4", "Content 4", ordinal);
```